homeassistant/components/binary_sensor/wink.py

```diff
@@ -0,0 +1,78 @@
+"""
+Support for Wink sensors.
+
+For more details about this platform, please refer to the documentation at
+at https://home-assistant.io/components/sensor.wink/
+"""
+import logging
+
+from homeassistant.components.binary_sensor import BinarySensorDevice
+from homeassistant.const import CONF_ACCESS_TOKEN, STATE_ON, STATE_OFF
+from homeassistant.helpers.entity import Entity
+
+REQUIREMENTS = ['python-wink==0.6.1']
+
+BINARY_TYPES = ['brightness',
```

Comment:
Can you provide an example of this? And what icons should I get? This is what they look like now.
![screenshot - 02232016 - 05 06 16 pm](https://cloud.githubusercontent.com/assets/6432770/13268310/d6777904-da4f-11e5-8792-cc8538580987.png)